bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java

```diff
@@ -216,21 +216,31 @@ protected synchronized void updateData() {
     }
 
     private boolean refreshProperties(MiIoBasicDevice device) {
+        MiIoCommand command = MiIoCommand.getCommand(device.getDevice().getPropertyMethod());
+        if (command == MiIoCommand.UNKNOWN) {
+            command = MiIoCommand.GET_PROPERTY;
+        }
+        int maxProperties = MAX_PROPERTIES;
+        if (device.getDevice().getMaxProperties() != 0) {
```

Comment:
Why would we not default it to 5 in the initialisation in DeviceMapping.java. Than we can avoid the check here

Reply:
Done